service/component-samples/owner/src/main/java/org/fido/iot/sample/OwnerServerApp.java

```diff
@@ -39,15 +39,23 @@ public static void main(String[] args) {
     System.setProperty(OwnerAppConstants.SERVER_PATH,
         Path.of(OwnerConfigLoader.loadConfig(OwnerAppConstants.SERVER_PATH)).toAbsolutePath()
             .toString());
-    
+
     Context ctx = tomcat.addContext("", null);
-    
+
     ctx.addParameter(OwnerAppConstants.DB_URL,
         OwnerConfigLoader.loadConfig(OwnerAppConstants.DB_URL));
     ctx.addParameter(OwnerAppConstants.DB_USER,
         OwnerConfigLoader.loadConfig(OwnerAppConstants.DB_USER));
     ctx.addParameter(OwnerAppConstants.DB_PWD,
         OwnerConfigLoader.loadConfig(OwnerAppConstants.DB_PWD));
+
+    ctx.addParameter("db.url",
+        OwnerConfigLoader.loadConfig(OwnerAppConstants.DB_URL));
+    ctx.addParameter("db.user",
+        OwnerConfigLoader.loadConfig(OwnerAppConstants.DB_USER));
+    ctx.addParameter("db.password",
+        OwnerConfigLoader.loadConfig(OwnerAppConstants.DB_PWD));
+
```

Comment:
Could you please help me understand why `db.url`, `db.user` and `db.password` properties were added when `OwnerAppConstants.DB_URL`, `OwnerAppConstants.DB_USER` and `OwnerAppConstants.DB_PWD` are already present?